src/Codec/Beam.hs

```diff
@@ -63,10 +66,11 @@ encode name =
 
 data Builder =
   Builder
-    { moduleName :: Term
+    { moduleName :: Operand
     , labelCount :: Word32
     , functionCount :: Word32
     , atomTable :: Map.Map BS.ByteString Int
+    , litTable :: [Literal]
```

Comment:
Prefer the full name if it's not too long. I think `literalTable` is not too much.